frontend/rolecall/src/app/performance/performance-editor.component.ts

```diff
@@ -364,7 +366,7 @@ export class PerformanceEditor implements OnInit, OnDestroy, AfterViewChecked {
   }
 
   onStep1Input(field: string, event: InputEvent) {
-    const targetValue = (event.target as any).value;
+    const targetValue = (event.target as HTMLInputElement).value;
```

Comment:
Just a note here that `event.target` might be null according to the type: `readonly target: EventTarget | null;`, so by forcing it into `HTMLInputElement`  there's theoretically a chance we'll hit a null value in this function at runtime.

Maybe just to be safe we can also add
```
if (!event.target) {
  return;
}
```

Reply:
OK.